app/src/main/res/layout/activity_main.xml

```diff
@@ -31,10 +31,4 @@
         android:text="@string/view_movies"
         android:onClick="launchMoviesActivity"/>
 
-    <Button
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:text="@string/search_history"
-        android:onClick="launchSearchHistoryActivity"/>
```

Comment:
some legacy redundancy 😅 There is no search history activity in the app and lab doesn't mention it. I deleted it to avoid further confusion.